docs/request-options.rst

```diff
@@ -803,7 +803,7 @@ Pass a string to specify a proxy for all protocols.
 
 .. code-block:: php
 
-    $client->request('GET', '/', ['proxy' => 'tcp://localhost:8125']);
+    $client->request('GET', '/', ['proxy' => 'http://localhost:8125']);
```

Comment:
Does the previous code still work too?

Reply:
`tcp://` *never* worked with cURL, only with the StreamHandler.

For the StreamHandler the `tcp://` version still works, because in `parse_proxy` I specifically check for the `http` scheme and return the URL as-is if it is not using the `http` scheme (https://github.com/guzzle/guzzle/pull/2850/files#diff-0cdc13d47562373b13de07ef6ea57235f3c5dc23a7cdf33fa47cd96c55ad8bacR424).

However it is not recommended to use `tcp://` variant any longer. It is worse in every regarding, because it does not support authentication and it is incompatible with cURL. That's why I adjusted the Docs.